CRM/Dedupe/Merger.php

```diff
@@ -1296,7 +1329,7 @@ public static function getRowsElementsAndInfo($mainId, $otherId) {
               $js,
             );
 
-            // Add the information to the migrationInfo (@todo Why?)
+            // Add the information to the migrationInfo
```

Comment:
Perhaps not, but the comments above were left in as well (like 'Add the element') so I left this in just to provide a 'header' really.